docs/release-process.md

```diff
@@ -0,0 +1,95 @@
+# Release Process
+
+## Determine the New Version
+Extension versions may not have suffixes like "-rc1".
+We are incrementing patch versions to differentiate release candidate builds.
+
+For example, the 1.0 release will start with a build at version 1.0.0.
+Version 1.0.0 is, at first, a release candidate.
+If changes are needed, we move to the next build at version 1.0.1.
+Version 1.0.1 is also a release candidate at first.
+If no changes are needed, 1.0.1 becomes the first public release of 1.0.
+
+## Set the New Version Throughout
+You must update:
+
+- `config.js`
+- `package.json`
+- `public/manifest.json`
+
+## Add a Change Log Section
+Here's the [Change Log](../CHANGELOG.md).
+If there is an "Unreleased changes" section, rename it to the new version.
+Do not include a "v" prefix in the heading.
+
+Add any other user-facing changes since the last release that aren't entered.
+Suggested sections:
+
+- Breaking changes
+- New features
+- Bug fixes
+- Little things
+- Documentation changes
+
+Items may include a link to GitHub issues if there is a relevant one, e.g. in bug fixes.
+We especially encourage linking externally reported issues.
+
+If you feel like it, move one Change Log item to a section on top titled "Spotlight change".
+
+## Propose the New Version
+Open a pull request (PR) for the version changes and Change Log update, ideally separate from
+any other changes.
+
+When this PR gets merged into `master`, the chosen version permanently associated with the merge
+commit.
+
+## Prepare an Unsigned Archive
+Clean the output directory, build, and package:
+
+```sh
+VERSION=<VERSION>
+rm -rf dist/
+yarn install --frozen-lockfile
+yarn buildProd
+zip -r oasis-wallet-$VERSION-$(git rev-parse --short HEAD).zip dist/
```

Comment:
```suggestion
zip -r oasis-wallet-$VERSION.zip dist/
```

Reply:
ok I'll remove that last line about re-uploading without the commit hash too

Reply:
Heads up: For Firefox you'll need `(cd dist && zip --recurse-paths ../oasis-wallet-$VERSION.zip ./)`

And it should work for Chrome too

Reply:
filed #196, thanks